src/main/java/com/owncloud/android/lib/resources/activities/model/Activity.java

```diff
@@ -31,12 +31,14 @@
 import java.util.Date;
 import java.util.List;
 
+import lombok.AllArgsConstructor;
 import lombok.Getter;
 
 /**
  * Activity Data Model
  */
 @Getter
+@AllArgsConstructor
```

Comment:
leads to new spotbugs warnings

Bug type EI_EXPOSE_REP2
In class com.owncloud.android.lib.resources.activities.model.ActivityIn  method new  com.owncloud.android.lib.resources.activities.model.Activity(int, Date,  Date, String, String, String, String, String, String, String, String,  String, String, String, List, RichElement)Field com.owncloud.android.lib.resources.activities.model.Activity.dateLocal variable named dateAt Activity.java:[line 41]

Bug type EI_EXPOSE_REP2
In class com.owncloud.android.lib.resources.activities.model.Activity
In method new com.owncloud.android.lib.resources.activities.model.Activity(int, Date, Date, String, String, String, String, String, String, String, String, String, String, String, List, RichElement)
Field com.owncloud.android.lib.resources.activities.model.Activity.datetime
Local variable named datetime
At Activity.java:[line 41]

Reply:
Ignore issue: https://github.com/nextcloud/android-library/pull/479/commits/f6d8bf032c22f9ae881a220979712fb3cadf9f3a